agent/util-scripts/pbench-collect-sysinfo

```diff
@@ -16,8 +16,10 @@ pbench_bin="`cd ${script_path}/..; /bin/pwd`"
 # Defaults
 group=default
 dir="/tmp"
+# array containing all the possible sysinfo options
 sysinfo_opts_default=( block libvirt kernel_config sos topology ara )
-sysinfo_opts=${sysinfo_opts_default[*]}
+# get comma seperated values
+sysinfo_comma_seperated_opts_default=$(IFS=,; echo "${sysinfo_opts_default[*]}")
```

Comment:
Spelling error: "separated"

You should call it either "comma_separated_sysinfo_opts_default" or "sysinfo_opts_default_comma_separated" to emphasize the relation between the two variables. I prefer the second one btw.

Reply:
Fixed.